src/main/java/LoginTest.java

```diff
@@ -35,12 +36,29 @@ void loginTest() throws InterruptedException {
                 .typePassword(password)
                 .clickLoginButton();
 
-        HomePage homePage = loginPage.clickLoginButton();
+        homePage = loginPage.clickLoginButton();
 
         homePage
                 .getHomepageMarker()
                 .shouldBe(visible);
 
-        Thread.sleep(5000);
+        Thread.sleep(3000);
+
+    }
+
+    @Test (priority = 1)
+    void configTest() throws InterruptedException {
+
+        ConfigPage configPage = PageFactory.openConfigPage();
+
+        configPage
+                .getConfigPageMarker()
+                .shouldBe(visible);
+
+        configPage.inputSaveCheckConfigValue(ConfigPage.getConfigTitleFieldTest());
```

Comment:
inputSaveCheckConfigValue - very unclear name. what do we do here? what is this action?